hack/run-e2e-test-kind.sh

```diff
@@ -4,11 +4,34 @@ here="$(dirname "$(readlink --canonicalize "${BASH_SOURCE[0]}")")"
 root="$(readlink --canonicalize "$here/..")"
 export SRIOV_NETWORK_OPERATOR_IMAGE="${SRIOV_NETWORK_OPERATOR_IMAGE:-sriov-network-operator:latest}"
 export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE="${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE:-origin-sriov-network-config-daemon:latest}"
+export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
+INTERFACES_SWITCHER="${INTERFACES_SWITCHER:-"test-suit"}"
+SUPPORTED_INTERFACE_SWTICHER_MODES=("test-suit", "system-service")
 RETRY_MAX=10
 INTERVAL=10
 TIMEOUT=300
 MULTUS_CNI_DS="https://raw.githubusercontent.com/intel/multus-cni/master/images/multus-daemonset.yml"
-test_pf_pci_addr="$1"
+
+while test $# -gt 0; do
+    case "$1" in
+    --device-netns-switcher)
+        INTERFACES_SWITCHER="$2"
+        if [[ ! "${SUPPORTED_INTERFACE_SWTICHER_MODES[@]}" =~ "${INTERFACES_SWITCHER}" ]]; then
+            echo "Error: unsupported interface switching mode: ${INTERFACES_SWITCHER}!"
+            echo "Supported modes are: ${SUPPORTED_INTERFACE_SWTICHER_MODES[@]}"
+            exit 1
+        fi
+        shift
+        shift
+        ;;
+    *)
+        if [[ -z "$test_pf_pci_addr" ]];then
```

Comment:
I would prefer to just have it as a flag like `device-netns-switcher` so we can easily extend in the future.
dont think backward compat is an issue here, worst case is minor update in test infra.

In any case, we can leave this as is and change later if needed.

Reply:
Added the flag, and also kept this part for backward compatibility since it will not affect functionality in any way, though the priority will be for what came later.